libraries/standard/json/src/json.c

```diff
@@ -836,6 +839,7 @@ static void skipObjectScalars( const char * buf,
  * @param[in] buf  The buffer to parse.
  * @param[in,out] start  The index at which to begin.
  * @param[in] max  The size of the buffer.
+ * @param[in] mode  The start marker of an array or object.
```

Comment:
```suggestion
 * @param[in] mode  The start marker of an array or object. Should be '[' or '{'.
```

Reply:
Seems a little contrary to the style of the other docstrings, but I don't mind. What do you think @dan4thewin?